app/ts/utils/augurContractUtils.ts

```diff
@@ -98,15 +99,14 @@ export const finalizeMarket = async (writer: AccountAddress, market: AccountAddr
 	})
 }
 
-export const derivePayoutDistributionHash = async (reader: AccountAddress, market: AccountAddress, payoutNumerators: EthereumQuantity[]) => {
-	// TODO, this can be computed locally too, see here: https://github.com/AugurProject/augur/blob/dev/packages/augur-core/src/contracts/Augur.sol#L243
-	const client = createReadClient(reader)
-	return await client.readContract({
-		abi: MARKET_ABI,
-		functionName: 'derivePayoutDistributionHash',
-		address: market,
-		args: [payoutNumerators]
-	})
+// see here: https://github.com/AugurProject/augur/blob/dev/packages/augur-core/src/contracts/Augur.sol#L243
+export const derivePayoutDistributionHash = (payoutNumerators: bigint[], numTicks: bigint, numOutcomes: bigint): `0x${ string }` => {
```

Comment:
Would be nice if we had a test suite that could test to make sure UI functions like this replicate the behavior of the contract functions they are mirroring.